styles/toolbox.css

```diff
@@ -0,0 +1,71 @@
+body {
```

Comment:
We should define this as a .less file and incorporate it into the theming rather than define colors here.

Reply:
Why don't we include this in the regular `.less` files and use `$extra_args['body_attributes']` to add the class to the `<body>` tag? We could drop this file altogether.